### PR TITLE
[14.0][ADD] account_invoice_report_utm_campaign : adds Campaign, Medium, and Source to analysis report

### PR DESCRIPTION
**Pull request by nayatec**
This module adds the Campaign, Medium, and Source fields to the Invoice Analysis report.

### CODE REVIEW

**Review by hparfr — APPROVED** [review]
LGTM thank

File: account_invoice_report_utm_campaign/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "Account Invoice Report UTM Campaign",
+    "summary": """
+    Add the Campaign, Medium, and Source fields to the Invoice Analysis report.
+    """,
+    "version": "14.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/account-invoice-reporting",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
```
Comment (hparfr) [inline account_invoice_report_utm_campaign/__manifest__.py:15]: you forget to add yourself as maintainer

**Review by bealdav — APPROVED** [review]
Nice contribution, works as expected. Thanks @nayatec

**Review by hparfr — APPROVED** [review]

File: account_invoice_report_utm_campaign/__manifest__.py (hunk 1 above)
Comment (hparfr) [reply account_invoice_report_utm_campaign/__manifest__.py:15]: (resolved)

**Comment by pedrobaeza** [comment]
Why it is called `account_invoice_report_utm_campaign` instead of just `account_invoice_report_utm`, adding the 3 UTM fields?

**Comment by nayatec** [comment]
Hello @pedrobaeza you have a point here... As a matter of fact i named it after another module we made with a similar name because we thought that "utm_campaign" helps to better contextualize the module. I'll rename it when proposing the migration.

**Pull request merged**